ledger/eval/cow.go

```diff
@@ -299,6 +301,8 @@ func (cb *roundCowState) commitToParent() {
 		cb.commitParent.mods.Txids[txid] = ledgercore.IncludedTransactions{LastValid: incTxn.LastValid, Intra: commitParentBaseIdx + incTxn.Intra}
 	}
 	cb.commitParent.txnCount += cb.txnCount
+	// no overflow because of 10B algo cap
```

Comment:
I could do this instead?
```
cb.commitParent.feesCollected.Raw = basics.AddSaturate(cb.commitParent.feesCollected.Raw, cb.feesCollected.Raw)
```

It's a wild case - a single block that overflows u64 with fees paid.  Even in a network with more than 10B algos, it seems impossible.  Surely the total supply is less than maxuint64, so this would mean _all_ the supply goes through fees, and in fact, more than once?  The only way that's imaginable is if the FeeSink itself did the recycling, yet that's impossible because because FeeSink can't spend any more, and fees paid by feesink don't get added to `feesCollected`.

I think I'd rather stick with the existing code but explain it better, so that readers don't worry about the 10B limit.  What do you think?

Reply:
I think it'll be better to improve the explanation like you proposed rather than changing the code to be a (bit) more complex.

Reply:
I explained why it can't overflow regardless of the 10B